lib/nvmf/rdma.c

```diff
@@ -826,6 +826,18 @@ spdk_nvmf_request_prep_data(struct spdk_nvmf_request *req)
 		req->xfer = spdk_nvme_opc_get_data_transfer(req->cmd->nvmf_cmd.fctype);
 	} else {
 		req->xfer = spdk_nvme_opc_get_data_transfer(cmd->opc);
+		if ((cmd->opc == SPDK_NVME_OPC_GET_FEATURES) ||
+		    (cmd->opc == SPDK_NVME_OPC_SET_FEATURES)) {
+			switch (cmd->cdw10 & 0xff) {
+				case SPDK_NVME_FEAT_LBA_RANGE_TYPE:
```

Comment:
Please align the case statements with the switch for consistency with the rest of the code.